README.md

```diff
@@ -1,4 +1,4 @@
-![coverage-shield-badge-1](https://img.shields.io/badge/coverage-74.7%25-yellow.svg)
+![coverage-shield-badge-1](https://img.shields.io/badge/coverage-74.92%25-yellow.svg)
 [![npm version](https://badge.fury.io/js/enketo-core.svg)](http://badge.fury.io/js/enketo-core) [![Build Status](https://travis-ci.com/enketo/enketo-core.svg?branch=master)](https://travis-ci.org/enketo/enketo-core) [![Dependency Status](https://david-dm.org/enketo/enketo-core/status.svg)](https://david-dm.org/enketo/enketo-core) [![devDependency Status](https://david-dm.org/enketo/enketo-core/dev-status.svg)](https://david-dm.org/enketo/enketo-core?type=dev) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/dc1c5aaa9267d75cbd2d6714d2b4fa32)](https://www.codacy.com/app/martijnr/enketo-core?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=enketo/enketo-core&amp;utm_campaign=Badge_Grade)
```

Comment:
This is fine, but for the future, I find it easier to leave this out and only update when we do a release to avoid out-of-order PR merge conflicts.

Reply:
That makes sense! I had assumed this was expected because it automatically updates on `npm test`. Okay if I add a note in tutorials/20-development.md?

Reply:
Yes, definitely. Thanks!